packages/mds-webpack-config/index.ts

```diff
@@ -62,12 +63,15 @@ const MergeConfigurations = (entry = 'index') => (config: CustomConfiguration):
           ] // TypeORM
         ].map(dependency => new IgnorePlugin(new RegExp(`^${dependency}$`))),
         // Make npm package name/version available to bundle
-        new DefinePlugin({
-          NPM_PACKAGE_NAME: JSON.stringify(npm_package_name),
-          NPM_PACKAGE_VERSION: JSON.stringify(npm_package_version),
-          NPM_PACKAGE_GIT_BRANCH: JSON.stringify(gitRevisionPlugin.branch()),
-          NPM_PACKAGE_GIT_COMMIT: JSON.stringify(gitRevisionPlugin.commithash()),
-          NPM_PACKAGE_BUILD_DATE: JSON.stringify(new Date().toISOString())
+        new WrapperWebpackPlugun({
```

Comment:
It's definitely supposed to be "Plugun" and not "Plugin"?

Reply:
It was a typo. It works because it's just a default export, but I fixed it. Thanks.